css/site.styl

```diff
@@ -196,7 +196,7 @@ strong
 
   // TEXT RULES
     
-.line-tighter
+.line-tighter, .line-tighter p
```

Comment:
This override on the `p` shouldnt be needed, but `p` elements have the `line-height` explicitly set even though they will inherit from body by default and thus shouldn't need the explicit value. 

However in the particular context of the citas text on the right hand column the containing div also has the `font-size-h4` class. This class has a size value set mobile first in the responsive styles, however in the `site.styl` `font-size-h4` sets a `line-height` value. I think that `line-height` is actually only wanted for h4 tags not the class too (the other heading matched to size classes do the same thing in `site.styl` but for bold etc)